modules/machineset-yaml-aws.adoc

```diff
@@ -184,28 +188,30 @@ $ oc -n openshift-machine-api \
     -o jsonpath='{.spec.template.spec.providerSpec.value.ami.id}{"\n"}' \
     get machineset/<infrastructure_id>-<role>-<zone>
 ----
-<5> Optional: Specify custom tag data for your cluster. For example, you might add an admin contact email address by specifying a `name:value` pair of `Email:\admin-email@example.com`.
-+
-[NOTE]
-====
-Custom tags can also be specified during installation in the `install-config.yml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yml` file.
-====
-
 ifndef::edge[]
-<6> Specify the zone, for example, `us-east-1a`.
+<5> Specify the zone, for example, `us-east-1a`.
 endif::edge[]
 ifdef::edge[]
-<6> Specify the zone name, for example, `us-east-1-nyc-1a`.
+<5> Specify the zone name, for example, `us-east-1-nyc-1a`.
 endif::edge[]
-<7> Specify the region, for example, `us-east-1`.
+<6> Specify the region, for example, `us-east-1`.
 ifndef::edge[]
+<7> Optional: Specify one or more additional security groups using the format shown.
+When you specify multiple security groups, the rules are merged and applied to the instance as described in link:https://docs.aws.amazon.com/vpc/latest/userguide/security-group-rules.html[{aws-short} documentation about security group rules].
+Refer to {aws-short} documentation for guidance on quotas and other limitations.
```

Comment:
I moved some other things around to fix numbering, but this one is the new content